src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -263,6 +266,9 @@ public void run() {
     }
 
     if (newVar != null) {
+      if (intoScalar) {
+        type = type.makeOptional();
```

Comment:
I would add an inline comment for clarity:
```
// "optional" is how protobuf defines singular fields
```

Reply:
Done